force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,34 @@
+trigger AccountTrigger on Account (before insert, after insert) {
+    if(Trigger.isBefore && Trigger.isInsert){
+        for(Account acc : Trigger.new){
+            if(String.isBlank(acc.Type)){
+                acc.Type = 'Prospect';
+            }    
+            if(!String.isBlank(acc.ShippingStreet) && !String.isBlank(acc.ShippingCity) && !String.isBlank(acc.ShippingState) && !String.isBlank(acc.ShippingPostalCode) && !String.isBlank(acc.ShippingCountry)){
+                acc.BillingStreet = acc.ShippingStreet; 
+                acc.BillingCity = acc.ShippingCity;
+                acc.BillingState = acc.ShippingState;
+                acc.BillingPostalCode = acc.ShippingPostalCode;
+                acc.BillingCountry = acc.ShippingCountry;
+            }
+            
+            if(String.isNotBlank(acc.Phone) && String.isNotBlank(acc.Fax) && String.isNotBlank(acc.Website)){
+            acc.Rating = 'Hot';
+            }
```

Comment:
If you can try to separate these different chunks of code so that you only test one block at a time. It's easier to troubleshoot failures that way